datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/manifest.py

```diff
@@ -0,0 +1,196 @@
+@click.argument('to_version', required=True)
+@click.pass_context
+def migrate(ctx, integration, to_version):
+    """
+    Helper tool to ease the migration of a manifest to a newer version, auto-filling fields when possible
+
+    Inputs:
+
+    integration: The name of the integration folder to perform the migration on
+
+    to_version: The schema version to upgrade the manifest to
+    """
+    echo_info(f"Migrating {integration} manifest to {to_version}....", nl=True)
+
+    # Perform input validations
+    if integration and integration not in get_valid_integrations():
+        abort(f'    Unknown integration `{integration}`')
+
+    loaded_manifest = JSONDict(load_manifest(integration))
+    manifest_version = loaded_manifest.get_path("/manifest_version")
+
+    if to_version not in SUPPORTED_MANIFEST_VERSIONS:
+        abort(f"    Unknown to_version: `{to_version}`")
+    if to_version == manifest_version:
+        abort(f"    {integration} is already on version `{manifest_version}`")
+    if to_version not in SUPPORTED_VERSION_UPGRADE_PATHS.get(manifest_version, []):
+        abort(f"    Can't migrate from version `{manifest_version}` to version: `{to_version}`")
+
+    migrated_manifest = JSONDict()
+    for key, val in V2_TO_V1_MAP.items():
+        if val == SKIP_IF_FOUND:
+            continue
+        elif isinstance(val, str):
+            final_value = loaded_manifest.get_path(val)
+        else:
+            final_value = val
+
+        if final_value is not None:
+            migrated_manifest.set_path(key, final_value)
+
+    # Update any previously skipped field in which we can use logic to assume the value of
+    # Also iterate through any lists to include new/updated fields at each index of the list
+    migrated_manifest.set_path("/manifest_version", "2.0.0")
+    migrated_manifest.set_path("/classifier_tags", TODO_FILL_IN)
+    migrated_manifest.set_path("/assets/integration/configuration/spec", TODO_FILL_IN)
+
+    # Retrieve and map all categories from other fields
+    classifier_tags = []
+    supported_os = loaded_manifest.get_path("/supported_os")
+    for os in supported_os:
+        classifier_tags.append(OS_TO_CLASSIFIER_TAGS.get(os.lower()))
+
+    categories = loaded_manifest.get_path("/categories")
+    for category in categories:
+        classifier_tags.append(CATEGORIES_TO_CLASSIFIER_TAGS.get(category.lower()))
+
+    # Write the manifest back to disk
+    migrated_manifest.set_path("/classifier_tags", classifier_tags)
+
+    # Marketplace-only fields:
+    if ctx.obj['repo_name'] == "marketplace":
+        migrated_manifest.set_path("/author/vendor_id", TODO_FILL_IN)
+        migrated_manifest.set_path("/author/sales_email", loaded_manifest.get_path("/terms/legal_email"))
+
+        migrated_manifest.set_path("/pricing", loaded_manifest.get_path("/pricing"))
+        migrated_manifest.set_path("/legal_terms", {})
+        migrated_manifest.set_path("/legal_terms/eula", "/terms/eula")
```

Comment:
This has the effect of setting the value to the literal `/terms/eula`.

Reply:
Good catch, I missed passing that into the loaded_manifest.get(), just pushed a new commit that fixes this